### PR TITLE
Add Ethereum Foundation page

### PR DESCRIPTION
**Pull request by samajammin**
## Description
Add Ethereum Foundation page

## Related Issue
#920 & #919

## Screenshots (if appropriate):
localhost_8888_foundation_

### CODE REVIEW

**Comment by jjmstark** [comment]
Awesome - single comment, would suggest re-phrasing the heading of the 2nd section to "Ethereum Foundation Initiatives"

File: docs/foundation/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <img :src="$withBase('/ef-logo.png')" class="ef-logo" />
+</div>
+
+The [Ethereum Foundation](http://ethereum.foundation/) (EF) is a non-profit organization dedicated to supporting [Ethereum](/what-is-ethereum/) and related technologies.
+
+The EF is not a company, or even a traditional non-profit. Their role is not to control or lead Ethereum, nor are they the only organization that funds critical development of Ethereum-related technologies. The EF is one part of a much larger [ecosystem](/community/).
+
+Learn more at [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.
+
+## Initiatives of the Ethereum Foundation
```
Comment (samajammin) [inline docs/foundation/index.md:26]:
```suggestion
## Ethereum Foundation Initiatives
```

**Comment by samajammin** [comment]
Great, thanks @jjmstark. FYI for future reference, a nifty GH feature: you can suggest changes while reviewing a PR that creates a suggested commit: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

**Comment by samajammin** [comment]
@ethjoe please review when possible 😄 - thanks!

**Comment by ethjoe** [comment]
I spoke with Josh via TG DM and suggested just a tiny change or two. I'm
heading to sleep now, but if the Learn More line was moved then I was all
set. Nice work.

File: docs/foundation/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+<div>
+  <img :src="$withBase('/ef-logo.png')" class="ef-logo" />
+</div>
+
+The [Ethereum Foundation](http://ethereum.foundation/) (EF) is a non-profit organization dedicated to supporting [Ethereum](/what-is-ethereum/) and related technologies.
+
+The EF is not a company, or even a traditional non-profit. Their role is not to control or lead Ethereum, nor are they the only organization that funds critical development of Ethereum-related technologies. The EF is one part of a much larger [ecosystem](/community/).
+
+Learn more at [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.
```
Comment (jjmstark) [inline docs/foundation/index.md:24]:
```suggestion
```

Let's move this bit to the very bottom of the page, in order to suggest to the reader that there are more EF initiatives than just the 2 listed here

File: docs/foundation/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Learn more about the Ecosystem Support Program, past grant recipients, and the grant application process at [ecosystem.support](https://ecosystem.support/). You can also view the [Ecosystem Support Program Blog](https://blog.ethereum.org/category/ecosystem-support-program/) for their latest news and announcements.
+
+### Devcon
+
+Since 2014, the Ethereum Foundation has organized Devcon, the annual conference for all Ethereum developers, researchers, thinkers, and makers.
+
+You can access video content of conference presentations for every year since its inception at [archive.devcon.org](https://archive.devcon.org/).
+
+Learn more at [devcon.org](https://devcon.org/) or check out the [Devcon Blog](https://blog.ethereum.org/category/devcon/) for the latest announcements.
```
Comment (jjmstark) [inline docs/foundation/index.md:40]:
```suggestion
Learn more at [devcon.org](https://devcon.org/) or check out the [Devcon Blog](https://blog.ethereum.org/category/devcon/) for the latest announcements.


Learn more at [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.
```

**Comment by ethjoe** [comment]
One other thing -- this also requires a small update to the Devcon line to
make sure that the last two lines aren't both "Learn More". That could just
mean something as simple as: "For more on the Foundation and their work..."

File: docs/foundation/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+
+Learn more at [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.
```
Comment (samajammin) [inline docs/foundation/index.md:42]:
```suggestion
For more on the Foundation and their work, visit [ethereum.foundation](http://ethereum.foundation/) or check out the [Ethereum Foundation Blog](https://blog.ethereum.org/) for the EF's latest news and announcements.
```

**Comment by samajammin** [comment]
@ethjoe updated to use the language you described 👍 - merging in!

**Comment by samajammin** [comment]
@all-contributors please add @ethjoe for content & review

**Pull request merged**